tasks/release/ReleaseVersionController.php

```diff
@@ -0,0 +1,81 @@
+<?php
+
+namespace MailPoetTasks\Release;
+
+require_once __DIR__ . '/Jira.php';
+
+class ReleaseVersionController {
+
+  /** @var Jira */
+  private $jira;
+
+  /** @var string */
+  private $project;
+
+  function __construct(Jira $jira, $project) {
+    $this->jira = $jira;
+    $this->project = $project;
+  }
+
+  static function createWithJiraCredentials($token, $user, $project) {
+    return new self(new Jira($token, $user, $project), $project);
+  }
+
+  function assignVersionToCompletedTickets($version) {
```

Comment:
Just an idea: What about making the version parameter optional and in case of `null` value it would automatically use latest version? `$this->jira->getVersion()` already returns latest version so we would just have to check whether is unreleased, which you already implemented 👍 .

Reply:
I like your idea, incorporated it.